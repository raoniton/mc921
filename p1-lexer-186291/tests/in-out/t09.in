class Main {
    public static void main(String[] args) {
        int n=1001, r = 0, t;

        t = n;

        while (t != 0)
        {
            r = r * 10;
            r = r + t%10;
            t = t/10;
        }

        if (n == r)
            print(n, " is a palindrome number.");
        else
            print(n, " isn't a palindrome number.");
    }
}