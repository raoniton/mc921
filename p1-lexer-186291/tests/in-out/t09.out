LexToken(CLASS,'class',1,0)
LexToken(ID,'Main',1,6)
LexToken(LBRACE,'{',1,11)
LexToken(PUBLIC,'public',2,17)
LexToken(STATIC,'static',2,24)
LexToken(VOID,'void',2,31)
LexToken(MAIN,'main',2,36)
LexToken(LPAREN,'(',2,40)
LexToken(STRING,'String',2,41)
LexToken(LBRACKET,'[',2,47)
LexToken(RBRACKET,']',2,48)
LexToken(ID,'args',2,50)
LexToken(RPAREN,')',2,54)
LexToken(LBRACE,'{',2,56)
LexToken(INT,'int',3,66)
LexToken(ID,'n',3,70)
LexToken(ASSIGN,'=',3,71)
LexToken(INT_LITERAL,'1001',3,72)
LexToken(COMMA,',',3,76)
LexToken(ID,'r',3,78)
LexToken(ASSIGN,'=',3,80)
LexToken(INT_LITERAL,'0',3,82)
LexToken(COMMA,',',3,83)
LexToken(ID,'t',3,85)
LexToken(SEMI,';',3,86)
LexToken(ID,'t',5,97)
LexToken(ASSIGN,'=',5,99)
LexToken(ID,'n',5,101)
LexToken(SEMI,';',5,102)
LexToken(WHILE,'while',7,113)
LexToken(LPAREN,'(',7,119)
LexToken(ID,'t',7,120)
LexToken(NE,'!=',7,122)
LexToken(INT_LITERAL,'0',7,125)
LexToken(RPAREN,')',7,126)
LexToken(LBRACE,'{',8,136)
LexToken(ID,'r',9,150)
LexToken(ASSIGN,'=',9,152)
LexToken(ID,'r',9,154)
LexToken(TIMES,'*',9,156)
LexToken(INT_LITERAL,'10',9,158)
LexToken(SEMI,';',9,160)
LexToken(ID,'r',10,174)
LexToken(ASSIGN,'=',10,176)
LexToken(ID,'r',10,178)
LexToken(PLUS,'+',10,180)
LexToken(ID,'t',10,182)
LexToken(MOD,'%',10,183)
LexToken(INT_LITERAL,'10',10,184)
LexToken(SEMI,';',10,186)
LexToken(ID,'t',11,200)
LexToken(ASSIGN,'=',11,202)
LexToken(ID,'t',11,204)
LexToken(DIVIDE,'/',11,205)
LexToken(INT_LITERAL,'10',11,206)
LexToken(SEMI,';',11,208)
LexToken(RBRACE,'}',12,218)
LexToken(IF,'if',14,229)
LexToken(LPAREN,'(',14,232)
LexToken(ID,'n',14,233)
LexToken(EQ,'==',14,235)
LexToken(ID,'r',14,238)
LexToken(RPAREN,')',14,239)
LexToken(PRINT,'print',15,253)
LexToken(LPAREN,'(',15,258)
LexToken(ID,'n',15,259)
LexToken(COMMA,',',15,260)
LexToken(STRING_LITERAL,'" is a palindrome number."',15,262)
LexToken(RPAREN,')',15,288)
LexToken(SEMI,';',15,289)
LexToken(ELSE,'else',16,299)
LexToken(PRINT,'print',17,316)
LexToken(LPAREN,'(',17,321)
LexToken(ID,'n',17,322)
LexToken(COMMA,',',17,323)
LexToken(STRING_LITERAL,'" isn\'t a palindrome number."',17,325)
LexToken(RPAREN,')',17,354)
LexToken(SEMI,';',17,355)
LexToken(RBRACE,'}',18,361)
LexToken(RBRACE,'}',19,363)
